### DISCUSSION TITLE
Can we obtain build information through resource version without using the website?

### DISCUSSION THREAD

**Discussion by PatShen** [concourse/concourse#8824] [Ideas]
I recently encountered a requirement that I need to obtain the information of builds which were triggered in a specific resource version. 
For now, I can find the builds information on the website(`{base_url}/teams/{team_name}/pipelines/{pipeline_name}/resources/{resource_name}`), but I can't find a way to get them by using `fly` or REST api. Do we have plans to add this feature?

**Comment by suhlig** [comment]
I think you are looking for [`fly curl`](https://concourse-ci.org/administration.html#fly-curl), e.g.

```command
$ fly curl /api/v1/builds
```

You'll need `--target`, too.
